terraform_validate.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Function to run terraform init and validate in a directory
+run_terraform() {
+  local dir="$1"
+  echo "Running terraform init and validate in $dir"
+  cd "$dir" || exit
+  terraform init
+  terraform validate
+  cd - || exit
```

Comment:
Maybe we should just set -x on this script instead of handling failures manually.